src/styles/blocks/catalog.scss

```
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

```suggestion
    transition-duration: 300ms;
    transition-property: color, transform;
```
Would be great to specify to which exactly properties you want to apply the transition. 
If not specified, it applies to all the properties which is not really good for performance.
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x

---

done